### ISSUE TITLE
VirusTotal has alert on last version

### ISSUE THREAD

**Issue by Alezy80** [d0vgan/AkelPad-Plugs-QSearch#1]
Last version has high detection rate on [VirusTotal](https://www.virustotal.com/gui/file/7a9bdf5bb7ecee573e1c4f3e23b644420aebfedf3c17aeea740e8f28a8d1f356), the previous version is clean

**Comment by d0vgan** [comment]
These all are false-positives and unfortunately I can do nothing with it. I published two different versions: QSearch-7.9.zip (built with Visual Studio 2005) and QSearch-7.9-vc2017.zip (built with Visual Studio 2017), but some anti-viruses detect false-positives in both:
https://github.com/d0vgan/AkelPad-Plugs-QSearch/releases/tag/v79
I reported these false-positives to a few anti-virus providers more than a month ago.
Probably the best we all can do with this situation is to report these false-positives to anti-virus providers to make them to react. The more false-positive reports, the higher are chances to fix it.
